EmaPajic/assignment3/assignment3.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+
+from dictionary import Dictionary
+
+def out_of_bounds(grid,i,j,n,m,visited):
+    #checks if coordinates are out of bounds or if it is already visited
+    if i < 0 or j < 0:
+        return True
+    if i >= n or j>= m:
+        return True
+    if visited[tuple((i,j))] == 1:
+        return True
+    return False
+
+def find_words_from_grid(grid,i,j,n,m,word,dictionary,foundWords,visited):
```

Comment:
Can you replace foundWords with found_words? The naming convention for function arguments in python is to have lowercase words separated by underscores.

Reply:
updated :)